assets/scss/views/_about.scss

```diff
@@ -36,14 +36,31 @@ main {
               @include flex-row;
 
               .member {
-                @include flex-row;
-
-                flex-basis: 50%;
-                flex-wrap: nowrap;
-                padding: 0 ($whitespace * 1.5) ($whitespace) 0;
+                border-radius: $radius;
+                display: flex;
+                flex-direction: row;
+                flex:  calc(50% - #{$whitespace});
+                margin: 0 $whitespace $whitespace 0;
+                padding: $whitespace;
+                transition: all $transition-duration $transition-timing-function;
```

Comment:
Can we only transition the properties that we actually need to transition please, to avoid additional compute/re-render overhead?